napari/components/dims.py

```diff
@@ -218,16 +218,33 @@ def _check_dims(cls, values):
         elif labels_ndim > ndim:
             updated['axis_labels'] = axis_labels[-ndim:]
 
+        # If the last used slider is no longer visible, use the first.
+        last_used = values['last_used']
+        ndisplay = values['ndisplay']
+        dims_range = values['range']
```

Comment:
Not certain, but I think you might want to use `range_` or equivalently `updated['range']` here instead, so we handle validation of multiple values on initialization.